### PR TITLE
serialization: add test, make volumes null

### PR DESCRIPTION
**Pull request by philips**
This is a spec change based on testing live Docker Hub configs. Volume may
be null as found in the hub.docker.com/library/docker image.

This is a continuation of https://github.com/opencontainers/image-spec/pull/145

Signed-off-by: Brandon Philips <brandon.philips@coreos.com>

### CODE REVIEW

**Comment by philips** [comment]
Please merge #145 first. I wanted to separate these out because this is a spec change to make the Volumes field OPTIONAL

File: schema/defs-config.json [hunk 1]
```diff
@@ -38,7 +38,10 @@
           }
         },
         "Volumes": {
-          "$ref": "defs.json#/definitions/mapStringObject"
+          "oneOf": [
+            {"$ref": "defs.json#/definitions/mapStringObject"},
+            {"type": "null"}
```
Comment (wking) [inline schema/defs-config.json]: This doesn't look like `jq` ;).  Did you run [`make fmt`][1] (with the fix from #148)?

[1]: https://github.com/opencontainers/image-spec/pull/148/files#diff-2cb561c5e46512465f54a0f84d421c80R59

**Comment by runcom** [comment]
LGTM (beside @wking nit) (IANAM)

**Comment by runcom** [comment]
This can be closed now - #152 has been merged with the commits from here
